tfx/examples/imdb/imdb_utils_native_keras.py

```diff
@@ -79,7 +85,7 @@ def preprocessing_fn(inputs):
   sentiment = inputs['sentiment']
   review = inputs['review']
   return {
-      'sentiment': _sentiment_to_int(sentiment),
+      'sentiment': sentiment,
       'embedding_input': _tokenize_review(review)}
 
 def _input_fn(file_pattern: List[Text],
```

Comment:
If we are to pad within the trainer (as suggested above), then in the _input_fn, set the batch_size in tf.data.experimental.make_batched_features_dataset to 1. Then after that, call dataset = dataset.padded_batch(batch_size). Finally, return dataset.

If we go down this path, we will also need to align the serving_input_fn.